### ISSUE TITLE
[UI Feedback] Keyboard shortcuts in annotation view

### ISSUE THREAD

**Issue by crosssenses** [pelagios/recogito2#617]
It would be very helpful to have keyboard shortcuts in annotation view to switch between the tools (move, point, …) to quickly add, edit and review annotations.

**Comment by rsimon** [comment]
Agreed & thx for the feedback - linking this to #482 and #483 for reference
